package.json

```diff
@@ -7,95 +7,91 @@
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@metamask/jazzicon": "^2.0.0",
-    "@mui/icons-material": "^5.2.1",
-    "@mui/material": "^5.4.0",
-    "@reduxjs/toolkit": "^1.4.0",
-    "@sentry/react": "^6.16.1",
-    "@sentry/tracing": "^6.16.1",
-    "@types/jest": "^26.0.22",
-    "@types/lodash": "^4.14.178",
-    "@types/node": "^14.14.35",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.7",
-    "@types/react-redux": "^7.1.11",
+    "@mui/icons-material": "^5.5.0",
+    "@mui/material": "^5.5.0",
+    "@reduxjs/toolkit": "^1.8.0",
+    "@sentry/react": "^6.18.2",
+    "@sentry/tracing": "^6.18.2",
+    "@types/jest": "^27.4.1",
+    "@types/lodash": "^4.14.179",
+    "@types/node": "^17.0.21",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.13",
+    "@types/react-redux": "^7.1.23",
     "@types/react-router-dom": "^5.3.3",
     "@uauth/web3-react": "^1.0.0",
     "@web3-react/abstract-connector": "^6.0.7",
     "@web3-react/core": "^6.1.9",
     "@web3-react/injected-connector": "^6.0.7",
-    "@web3-react/network-connector": "^6.1.9",
+    "@web3-react/network-connector": "^6.2.9",
     "@web3-react/types": "^6.0.7",
     "@web3-react/walletconnect-connector": "^6.2.13",
-    "@web3-react/walletlink-connector": "^6.2.5",
-    "async-retry": "^1.3.1",
+    "@web3-react/walletlink-connector": "^6.2.13",
+    "async-retry": "^1.3.3",
     "classnames": "^2.3.1",
     "copy-to-clipboard": "^3.3.1",
-    "ethcall": "^3.2.0",
-    "ethers": "^5.5.3",
-    "framer-motion": "^4.0.3",
-    "history": "^5.0.0",
-    "i18next": "^19.8.3",
-    "i18next-browser-languagedetector": "^6.0.1",
-    "i18next-fetch-backend": "^3.0.0",
-    "lodash": "^4.17.20",
-    "react": "^17.0.1",
-    "react-device-detect": "^1.14.0",
-    "react-dom": "^17.0.1",
-    "react-i18next": "^11.11.0",
-    "react-redux": "^7.2.4",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
-    "react-toastify": "^8.1.1",
+    "ethcall": "3.2.0",
```

Comment:
we omitting this upgrade for now?

Reply:
Yes I'll most likely follow up on it in a separate PR